book.common/src/main/java/com/happystays/book/common/dto/CreditCard.java

```diff
@@ -0,0 +1,14 @@
+package com.happystays.book.common.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class CreditCard {
+    private String maskedCardNumber ;
+    private String cardHolderFirstName;
+    private String cardHolderLastName;
```

Comment:
no need of first name last name.. just cardsHoldername is enough..need to change the schema